lua/octo/reviews/init.lua

```diff
@@ -51,6 +51,36 @@ function Review:create(callback)
   }
 end
 
+-- Get review threads without start a review.
+function Review:populate_threads(callback)
+  local query =
+    graphql("review_threads_query", self.pull_request.owner, self.pull_request.name, self.pull_request.number)
+  -- print query to file /tmp/query
+  local file = io.open("/tmp/query", "w")
+  file:write(query)
+  file:close()
+
+  gh.run {
+    args = { "api", "graphql", "-f", string.format("query=%s", query) },
+    cb = function(output, stderr)
+      if stderr and not utils.is_blank(stderr) then
+        utils.error(stderr)
+      elseif output then
+        local resp = vim.fn.json_decode(output)
+        callback(resp)
+      end
+    end,
+  }
```

Comment:
Interested in using the new pattern?  

```lua
gh.api.graphql {
    query = query,
    opts = {
        cb = cb,
    }, 

}
```

Reply:
Definitely, that is very nice. Will update.

Reply:
ah I forgot about this and your other comment. will address this in a bit

Reply:
I updated this bit of code. Nice work on that abstraction, I like it :)